autopilot/hostfilter.go

```diff
@@ -52,26 +52,23 @@ func isUsableHost(cfg Config, gs bus.GougingSettings, rs bus.RedundancySettings,
 
 // isUsableContract returns whether the given contract is usable and whether it
 // can be renewed, along with a list of reasons why it was deemed unusable.
-func isUsableContract(cfg Config, h Host, c worker.Contract, bh uint64) (bool, bool, []string) {
-	var reasons []string
-	renewable := true
-
+func isUsableContract(cfg Config, h Host, c worker.Contract, bh uint64) (usable bool, refresh bool, renew bool, reasons []string) {
 	if isOutOfFunds(cfg, h, c) {
 		reasons = append(reasons, "out of funds")
+		refresh = true
 	}
 	if isUpForRenewal(cfg, c.Revision, bh) {
 		reasons = append(reasons, "up for renewal")
+		renew = true
```

Comment:
Wouldn't a renew take priority over a refresh? If so should we set `refresh = false` here? A contract should never show up for both a refresh and renew.